tests/averaging/rotation/test_rotation_averaging_base.py

```diff
@@ -22,6 +23,54 @@ def setUp(self):
 
         self.obj = DummyRotationAveraging()
 
+    def test_circle_two_edges(self):
+        """Tests for 4 poses in a circle, with a pose connected to its immediate neighborhood."""
+
+        if isinstance(self.obj, DummyRotationAveraging):
+            self.skipTest("Test case invalid for dummy class")
+
+        i2Ri1_dict, wRi_expected = sample_poses.convert_data_for_rotation_averaging(
+            sample_poses.CIRCLE_TWO_EDGES_GLOBAL_POSES, sample_poses.CIRCLE_TWO_EDGES_RELATIVE_POSES
+        )
+        wRi_computed = self.obj.run(len(wRi_expected), i2Ri1_dict)
+        self.assertTrue(geometry_comparisons.compare_rotations(wRi_computed, wRi_expected))
+
+    def test_circle_all_edges(self):
+        """Tests for 4 poses in a circle, with a pose connected all others."""
```

Comment:
Do you mean "Test"? Because there is only a single test in this method.

Reply:
yep. Thanks